README.md

```diff
@@ -265,6 +266,28 @@ These variables must follow a specific naming convention, as listed in the [Boto
 
 A list of all locations checked for Amazon Web Services credentials can also be found [here](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/credentials.html#configuring-credentials).
 
+### Neo4J
+
+Change the configuration using `%%graph_notebook_config` and modify the fields for `host`, `port`, `ssl`, and `neo4j` authentication. For a local Neo4j Desktop database, if querying via Bolt protocol, you can use the following command:
+
+```
+%%graph_notebook_config
+{
+  "host": "localhost",
+  "port": 7687,
+  "ssl": false,
+  "neo4j": {
+    "username": "neo4j",
+    "password": "password",
+    "auth": true
+  }
+}
+```
+
+Ensure that you also specify the `%%oc bolt` option when submitting queries to the Bolt endpoint. Likewise, do not specify the `bolt` option if you are querying the HTTP endpoint(`"port": 7474`).
```

Comment:
There is no HTTP endpoint for Neo4j so we should not highlight this and just say always use `bolt`

Reply:
Thanks for pointing this out. While Neo4j supports an [HTTP API](https://neo4j.com/docs/http-api/current/introduction/#http-api-introduction), this appears to not be supported on Neo4j's main service AuraDB, so I will remove the docs regarding HTTP endpoint. The related client code for Neo4J HTTP connections will be kept, in case a user wants to query a Neo4J db that does support the HTTP API (e.g. local instance created from Neo4J Desktop).